include/rtc/rtc.h

```diff
@@ -77,6 +77,28 @@ typedef enum { // Don't change, it must match plog severity
 	RTC_LOG_VERBOSE = 6
 } rtcLogLevel;
 
+#if RTC_ENABLE_MEDIA
+
+typedef enum {
+    // video
+    RTC_H264,
```

Comment:
Other enums have an explicit type in their name so this should be `RTC_CODEC_H264` for consistency.

Reply:
Done